modules/ROOT/pages/push/admin-ui.adoc

```diff
@@ -0,0 +1,92 @@
+
+=== Login and landing page
+
+Once you have the Unified Push Server running, click on it's route in OpenShift and login with your OpenShift credentials. You will see one PushApplication already created. The PushApplication is a logical construct, that represents an overall mobile application container for its _different_ mobile platforms, which are called Variants.
+
+The _Variants_ tab of the PushApplication gives a list of all existing variants. It shows details about the variant, such as the number of sent messages and registered devices. It also contains more details about the involved 3rd party Push Network.
+
+=== Managing registered device
+
+The details view of a specific variant shows a list of the registered devices at the bottom: 
+
+image::ups-admin-devices1.png[Device list]
+
+The table shows the metadata of each device, including the device token, subscribed categories or - if present - the used alias. On this screen you could also exclude a specific device from receiving Push Notifications, by unchecking the _Receiving_ check-box.
+
+=== Importing Device Metadata
+
+For various reasons, e.g. migrating from a custom solution to the Unified Push Server, it is possible to import devices and their metadata, including their devicetoken, into a variant. This is possible via cURL and on the UI. For that, on a specific variant, click on the _More_ menu you have the choice to select the _Import_ option.  
+
+The following dialog allows you to upload a JSON file, containing the device metadata.
+
+NOTE: The API for the JSON importer is described link:https://www.aerogear.org/docs/specs/aerogear-unifiedpush-rest-1.0.x/registry/device/importer/index.html#POST[here].
+
+=== Exporting Device Metadata
+
+Similar, the Unified Push Server has an option to export the device metadata of a variant. This is handy when you want to move from one Unified Push Server instance to a different one. On the _More_ menu of a specific variant, you have the choice to select the _Export_ option.
+
+Starting the export will generate a JSON file, which is being downloaded to your machine.
+
+=== Sending a Push Notification from code
+
+While sending a Push Notification from the AdminUI is a nice feature, in a real world scenario, the Push Notification is triggered by a backend, as explained in link:#_server_integration_tutorials[Server Integration Tutorials].
+
+The Unified Push Server comes with APIs for link:../../../unifiedpush/GetStartedwithJavaSender/[Java] and link:https://github.com/aerogear/aerogear-unifiedpush-nodejs-client#examples[Node.js]. Due to its RESTful architecture any backend, written in any language that supports HTTP(S), can link:../../../specs/aerogear-unifiedpush-rest/sender/index.html[send Push Notification requests] to it. On the details page of a PushApplication you find the required +Server URL+ and credentials (+Application ID+ and +Master Secret+).
```

Comment:
Are those links still up to date? We should probably avoid describing the backend libs because we haven't tested them in the context of this release. What do you think @finp ?